force-app/main/default/classes/CON_ContactMerge.cls

```diff
@@ -36,6 +36,28 @@
 * @group-content ../../ApexDocContent/ContactMerge.htm
 */
 public class CON_ContactMerge {
+
+    @TestVisible
+    private ContactSelector contactSelector {
```

Comment:
The class names AccountSelector and ContactSelector should be reserved for potential use at the adapter or domain layers. Any queries factored out of these classes related to contact merging should be added to the ContactMergeSelector that is also introduced here in this PR.